impl/src/test/groovy/io/jsonwebtoken/impl/DefaultJwtParserBuilderTest.groovy

```diff
@@ -92,4 +96,25 @@ class DefaultJwtParserBuilderTest {
             assertEquals DefaultJwtParserBuilder.MAX_CLOCK_SKEW_ILLEGAL_MSG, expected.message
         }
     }
+
+    @Test
+    void testDefaultDecoder() {
```

Comment:
Really minor, but unlike the test methods above which actually set a `Decoder`, this one sets a `Deserializer`
```suggestion
    void testDefaultDeserializer() {
```

Reply:
good catch, copypasta